packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## v4.0.0-extensibility-preview.4 (Feb 12, 2025)
 - Replace `event-emitter` in favor of the native `EventTarget` [#2289](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2289)
+- Call app extension's new methods `getRoutes` and `getRoutesAsync` [#2308](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2308)
+- Call app extension's `beforeRouteMatch` method with updated parameters [#2308](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2308)
```

Comment:
NIT: You have 2 entries here and you are using the word "Call" differently in both each context, the top you were are using "call" and it's name change, and the bottom one, "call" as in calling a function/invoking.

Reply:
Nope, I meant the same thing, as in calling the function. In the route-component and react-rendering, our React SDK calls those methods (getRoutes, getRoutesAsync, and beforeRouteMatch).